src/js/components/DateInput/stories/RangeFormat.js

```diff
@@ -26,9 +26,12 @@ export const RangeFormat = () => {
             value={value}
             format="mm/dd/yyyy-mm/dd/yyyy"
             buttonProps={{
-              label: `${dateFormat.format(
-                new Date(value[0]),
-              )} - ${dateFormat.format(new Date(value[1]))}`,
+              label:
```

Comment:
I deleted the "31" in the first date and saw this:

<img width="1442" alt="Screen Shot 2021-06-21 at 4 36 15 PM" src="https://user-images.githubusercontent.com/12522275/122840402-dedb7980-d2ae-11eb-98c7-8bf82350d7c8.png">

Also -- are `buttonProps` doing anything in this file? The docs seems to indicate that this prop is only relevant when there is no `format`

Reply:
good catches, both.
I've updated that story to not use `buttonProps`.
Also, I've fixed the stack trace issue.